package.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "@digitalbazaar/ecdsa-multikey",
+  "version": "1.0.0",
```

Set to pre-1.0.0 so our release tooling increments it properly.
```suggestion
  "version": "0.0.1",
```